python_modules/libraries/dagster-k8s/dagster_k8s/client.py

```diff
@@ -133,6 +133,55 @@ def k8s_api_retry(
     check.failed("Unreachable.")
 
 
+def k8s_api_retry_creation_mutation(
+    fn: Callable[..., None],
+    max_retries: int,
+    timeout: float,
+    msg_fn=lambda: "Unexpected error encountered in Kubernetes API Client.",
+) -> None:
+    check.callable_param(fn, "fn")
+    check.int_param(max_retries, "max_retries")
+    check.numeric_param(timeout, "timeout")
+
+    remaining_attempts = 1 + max_retries
+    retry_count = 0
+    while remaining_attempts > 0:
+        remaining_attempts -= 1
+
+        try:
+            fn()
+            return
+        except kubernetes.client.rest.ApiException as e:
+            retry_count = retry_count + 1
+            # Only catch whitelisted ApiExceptions
+            status = e.status
+
+            if retry_count > 1 and status == 409:
```

Comment:
doc block / extra comment on this 409 line to make it stand out what the difference is between this and the other